libsolidity/parsing/Parser.cpp

```diff
@@ -112,8 +112,16 @@ ASTPointer<SourceUnit> Parser::parse(shared_ptr<Scanner> const& _scanner)
 				nodes.push_back(parseFunctionDefinition(true));
 				break;
 			default:
+				if (
+					// Workaround because `error` is not a keyword.
+					m_scanner->currentToken() == Token::Identifier &&
+					currentLiteral() == "error" &&
```

Comment:
Shouldn't this be in the `Token` enum? Or would that automatically make it a keyword?

Reply:
It would make it a keyword.

Reply:
ok